package/crossplane.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: meta.pkg.crossplane.io/v1alpha1
+kind: Provider
+metadata:
+  name: provider-gcp
+spec:
+  controller:
+    image: crossplane/provider-gcp-controller:master
```

Comment:
This is kind of tricky since we are building the image for the controller as the same time, but we manually tag in integration tests so we can pin this on release branches and it should work as expected. However, non-releases will basically break as soon as a new version comes out on `master`. I am wondering if we will want to incorporate overriding this using Crossplane CLI during provider package build. We could also solve this now by making the build process replace this using `sed` or other.

Reply:
Keeping in draft until this is resolved because I don't think we are going to want to build with `master` here obviously, but want to give folks an opportunity to weigh in on if just doing a replace in the `build` pipeline is enough for now.

Reply:
> However, non-releases will basically break as soon as a new version comes out on master.

Can you elaborate a little - I'm not following why non-releases will break.

Reply:
@negz if we tag the controller image as `master` in the package image, then a new version of the controller image gets pushed, then old package images will point to a different controller image than they were "originally built" with.